src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/persistence/SQLitePersistor.java

```diff
@@ -321,8 +376,10 @@ private synchronized JsonElement getNonTemperatureRcas(String rca) {
         }
       }
     } catch (DataAccessException de) {
-      // it is totally fine if we fail to read some certain tables.
-      LOG.warn("Fail to read RCA : {}, query = {},  exceptions : {}", rca, rcaQuery.toString(), de.getStackTrace());
+      if (!de.getMessage().contains("no such table")) {
+        // it is totally fine if we fail to read some certain tables.
+        LOG.warn("Fail to read RCA : {}.", rca, de);
+      }
```

Comment:
should we print an error log if otherwise

Reply:
You mena make it an error log instead of warning ?